### PR TITLE
Tests for stats calculator

### PR DESCRIPTION
**Pull request by psrpinto**
This PR adds tests for the stats calculation, i.e. it tests that the table that is shown in the event page displays correct data.

### CODE REVIEW

File: tests/stats-calculator.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+		$user1_id = 42;
+		$user2_id = 43;
+		$user3_id = 44;
+
+		$event1_id = $this->event_factory->create_active( array( $user1_id ) );
+		$event2_id = $this->event_factory->create_active( array( $user1_id ) );
+
+		// For event1, aa locale, multiple users.
+		$this->stats_factory->create( $event1_id, $user1_id, 11, 'create' );
+		$this->stats_factory->create( $event1_id, $user2_id, 11, 'create' );
```
Comment (akirk) [inline tests/stats-calculator.php]: How could a user create two translations with the same id? I understand it's possible from a data perspective but it should not happen in reality.

**Review by akirk — COMMENTED** [review]
Looks good but not sure about one test.

File: tests/stats-calculator.php (hunk 1 above)
Comment (psrpinto) [reply tests/stats-calculator.php]: Makes sense, fixed.

**Review by akirk — APPROVED** [review]
Thank you!

**Pull request merged**